website/config/algolia/crawler-settings.js

```diff
@@ -13,23 +13,29 @@ new Crawler({
       recordExtractor: ({ $, helpers }) => {
         // priority order: deepest active sub list header -> navbar active item -> 'Documentation'
         const lvl0 =
-          $(
-            ".menu__link.menu__link--sublist.menu__link--active, .navbar__item.navbar__link--active"
-          )
-            .last()
-            .text() || "Documentation";
+          $(".navbar__item.navbar__link--active").last().text() ||
```

Comment:
The menu selector that came with the default setup was doing really weird things, and we prefer to just use "For users" vs "For contributors" as the lvl0 of the hierarchy so that we can use it for search faceting, eg to downrank contributor docs when viewing user docs and vice-versa